tooling/sync/README.md

```diff
@@ -43,7 +50,7 @@ It's advisable to only run flamegraphs on blocks that have already been synced,
 
 - `make flamegraph-main` and `make flamegraph-branch` can be used to run benchmarks on the main branch of the repo or a custom branch, respectively; generating both a flamegraph and logs of the run. `NETWORK` and `SYNC_BLOCK_NUM` must be provided, `EVM` can be optionally provided too. `BRANCH` must be provided for `flamegraph-branch` as well. `make flamegraph` can also be used as a branch agnostic option.
 
-- `make start-lighthouse` can be used to start lighthouse. `NETWORK` must be provided.
+- `make start-lighthouse` can be used to start lighthouse. `NETWORK` must be provided or else mainnet will be used as default.
```

Comment:
Is this desirable? I tend to default things to a staging environment, e.g. localnets or testnets like Holesky. That way you can't start spending real money by accident.

Reply:
IMO it's a default as good as any other. Running CL+EL on mainnet doesn't lead to loss of funds unless ETH is staked, which isn't expected for interacting with dev tooling.